content/docs/addons-test-utils.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: test-utils
-title: Test Utilities
+title: Test Araçları
 permalink: docs/test-utils.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**Ekleme**
 
 ```javascript
 import ReactTestUtils from 'react-dom/test-utils'; // ES6
-var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
+var ReactTestUtils = require('react-dom/test-utils'); // ES5 npm ile
 ```
 
-## Overview {#overview}
+## Test Araçlarına Genel Bakış {#overview}
 
-`ReactTestUtils` makes it easy to test React components in the testing framework of your choice. At Facebook we use [Jest](https://facebook.github.io/jest/) for painless JavaScript testing. Learn how to get started with Jest through the Jest website's [React Tutorial](https://jestjs.io/docs/tutorial-react).
+`ReactTestUtils`, React bileşenlerini seçtiğiniz test çerçevesinde test etmeyi kolaylaştırır. Facebook, JavaScript testi için Jest‘i kullanmaktadır. [Buradan](http://facebook.github.io/jest/docs/en/tutorial-react.html#content) React ile Jest'e nasıl başlayacağınız hakkında bilgi bulabilirsiniz.
```

Comment:
`Jest sitesindeki [React Tutorial](http://facebook.github.io/jest/docs/en/tutorial-react.html#content) ile Jest'e nasıl başlayacağınızı öğrenin`

I think we can skip translating the tutorial name here as we're not aware if Jest website is translated or not (probably not). This way the name here and on that website will be consistent at least. What do you think?

Reply:
Done.